### DISCUSSION TITLE
Create "official" docker image for server

### DISCUSSION THREAD

**Discussion by GhostDog98** [redeclipse/base#1545] [Ideas and Feature Requests]
Hi, recently got into this game and could see some advantages of offering a docker image for running a server, specifically:

- Less issues server side due to consistent and standardized environments
- Higher security due to containerized approach, leaving less possibility for a security issue to become a larger issue like RCE/ACE
- Easier to approach starting a server

If this is something others want, I could look into making it myself, but wanted to gauge actual interest before doing so lol

**Comment by dc-redeclipse** [comment]
Hallo and welcome to our Red Eclipse Community.
We already have a working docker image and I personally run servers with it.
The reasons why it's not in the docs in details is the very very low demand for it. If you like to set up one with docker we will help you out.

**Reply by GhostDog98** [reply]
I've got a fair amount of familiarity with docker already, so i should be fine, just couldn't find it anywhere lol
I'm guessing the `iceflower/redeclipse-server` package on docker hub is the one?

**Reply by dc-redeclipse** [reply]
> I've got a fair amount of familiarity with docker already, so i should be fine, just couldn't find it anywhere lol
> 
> I'm guessing the `iceflower/redeclipse-server` package on docker hub is the one?

Yes exactly!
Everything else you need is here:
[https://www.redeclipse.net/docs/server-how-to/Server-Setup](https://www.redeclipse.net/docs/server-how-to/Server-Setup)
